pkg/cmd/roachtest/cluster.go

```diff
@@ -2417,7 +2417,9 @@ func (c *clusterImpl) RunE(ctx context.Context, options install.RunOptions, args
 	defer l.Close()
 
 	cmd := strings.Join(args, " ")
-	c.f.L().Printf("running cmd `%s` on nodes [%v]; details in %s.log", roachprod.TruncateString(cmd, 30), nodes, logFile)
+	if c.l.File != nil {
+		c.f.L().Printf("running cmd `%s` on nodes [%v]; details in %s.log", roachprod.TruncateString(cmd, 30), nodes, logFile)
```

Comment:
I think we should still print the `running cmd` part if the file is nil, just not the second half of the line.